libvast/src/format/json.cpp

```diff
@@ -1053,7 +1054,7 @@ add(const ::simdjson::dom::object& object, table_slice_builder& builder) {
               const auto added = builder.add(caf::none);
               VAST_ASSERT(added);
             } else {
-              add(element, field.type, builder);
+              [[maybe_unused]] auto err = add(element, field.type, builder);
```

Comment:
We shouldn't be able to run into this error, as that would be a precondition violation. Please check `!err` with an assertion.